<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Shapes</class>
 <widget class="QMainWindow" name="Shapes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="RenderArea" name="renderArea" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="btnBackground">
          <property name="text">
           <string>Background Color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLineColor">
          <property name="text">
           <string>Line Color</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btnAstroid">
          <property name="text">
           <string>Astroid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCycloid">
          <property name="text">
           <string>Cycliod</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnHuygens">
          <property name="text">
           <string>Huygens</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnHypo">
          <property name="text">
           <string>Hypo Cycloid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLine">
          <property name="text">
           <string>Line</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCircle">
          <property name="text">
           <string>Circle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEllipse">
          <property name="text">
           <string>Elipse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnFancy">
          <property name="text">
           <string>Fancy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnStarfish">
          <property name="text">
           <string>StarFish</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCloud">
          <property name="text">
           <string>Colud 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCloud2">
          <property name="text">
           <string>Cloud 2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Scale :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spinScale">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>200.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Interval Length :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spinInterval">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Step Count :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinCount">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>512</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>575</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RenderArea</class>
   <extends>QWidget</extends>
   <header>renderarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
